src/explore-education-statistics-admin/src/pages/publication/components/__tests__/PublicationForm.test.tsx

```diff
@@ -148,10 +150,33 @@ describe('PublicationForm', () => {
         }),
       ).toBeInTheDocument();
 
+      // NOTE: Contact telelphone number is optional, so no validation error
+    });
+  });
+
+  test('show validation error when contact tel no is not valid', async () => {
```

Comment:
Same as `PublicationContactForm`, we should create more test cases to cover the various validation permutations.